ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuiteRunner.scala

```diff
@@ -129,7 +134,13 @@ final class LedgerTestSuiteRunner(
         completionCallback(result)
         ec.shutdown()
       }
+  }
 
+  def verifyRequirementsAndRun(completionCallback: Try[Vector[LedgerTestSummary]] => Unit): Unit = {
+    verifyRequirements.fold(
+      throwable => completionCallback(throwable),
```

Comment:
does this compile? `throwable` is of type `Throwable` and ` completionCallback` expects a `Try[...]`?

Reply:
It doesn't. :laughing:
I was caught off guard by a false positive by the IDE and pushed the code without testing locally.
https://github.com/digital-asset/daml/pull/2721/commits/1ce3bfd229af55fe831d74b8909f673d775ee83c addresses this.

Reply:
I know the feeling ;)